# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Hajime Taira <htaira@redhat.com>, 2011
# Satoshi Matsumoto <kaorimatz@gmail.com>, 2014
# Tadashi Jokagi <elf@elf.no-ip.org>, 2011
# takayuki988 <takayuki988@star.odn.ne.jp>, 2014
msgid ""
msgstr ""
"Project-Id-Version: dnf\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-04-25 17:37+0200\n"
"PO-Revision-Date: 2014-04-25 15:37+0000\n"
"Last-Translator: jsilhan <jsilhan@redhat.com>\n"
"Language-Team: Japanese (http://www.transifex.com/projects/p/dnf/language/ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: ../dnf/persistor.py:161
msgid "Invalid groups database, clearing."
msgstr ""

#: ../dnf/persistor.py:180
msgid "Unsupported installed groups database found, resetting."
msgstr ""

#: ../dnf/persistor.py:243
#, python-format
msgid "%s is empty file"
msgstr ""

#: ../dnf/drpm.py:56 ../dnf/repo.py:289
#, python-format
msgid "unsupported checksum type: %s"
msgstr ""

#: ../dnf/drpm.py:127
msgid "Delta RPM rebuild failed"
msgstr ""

#: ../dnf/drpm.py:129
msgid "Checksum of the delta-rebuilt RPM failed"
msgstr ""

#: ../dnf/comps.py:348
#, python-format
msgid "Environment '%s' is already installed."
msgstr ""

#: ../dnf/comps.py:368 ../dnf/comps.py:390
#, python-format
msgid "Environment '%s' is not installed."
msgstr ""

#: ../dnf/comps.py:415
#, python-format
msgid "Group '%s' is already installed."
msgstr ""

#: ../dnf/comps.py:430 ../dnf/comps.py:445
#, python-format
msgid "Group '%s' not installed."
msgstr ""

#: ../dnf/repo.py:374
#, python-format
msgid "Failed to synchronize cache for repo '%s': %s"
msgstr ""

#: ../dnf/yum/rpmtrans.py:107
msgid "Cleanup"
msgstr "整理中"

#: ../dnf/yum/rpmtrans.py:108 ../dnf/cli/output.py:1258
msgid "Downgrading"
msgstr "ダウングレード中"

#: ../dnf/yum/rpmtrans.py:109
msgid "Erasing"
msgstr "削除中"

#: ../dnf/yum/rpmtrans.py:110 ../dnf/cli/output.py:1254
msgid "Installing"
msgstr "インストールしています"

#: ../dnf/yum/rpmtrans.py:111 ../dnf/cli/output.py:1961
msgid "Obsoleting"
msgstr "不要削除"

#: ../dnf/yum/rpmtrans.py:112 ../dnf/cli/output.py:1256
msgid "Reinstalling"
msgstr "再インストール中"

#: ../dnf/yum/rpmtrans.py:113 ../dnf/cli/output.py:1255
msgid "Upgrading"
msgstr ""

#: ../dnf/yum/history.py:759
#, python-format
msgid "Unable to initialize yumdb history: %s"
msgstr ""

#: ../dnf/yum/history.py:1676
#, python-format
msgid "Can not create history database at '%s'."
msgstr ""

#: ../dnf/cli/option_parser.py:50
#, python-format
msgid "Command line error: %s"
msgstr "コマンドライン エラー: %s"

#: ../dnf/cli/option_parser.py:106
msgid "--color takes one of: auto, always, never"
msgstr "--color がとることができる値な次のうちひとつです: auto、always、never"

#: ../dnf/cli/option_parser.py:140
#, python-format
msgid "Options Error: %s"
msgstr "オプションエラー: %s"

#: ../dnf/cli/option_parser.py:151
#, python-format
msgid "--installroot must be an absolute path: %s"
msgstr "--installroot は絶対パスでなければなりません: %s"

#: ../dnf/cli/option_parser.py:168
msgid "allow erasing of installed packages to resolve dependencies"
msgstr ""

#: ../dnf/cli/option_parser.py:171
msgid "try the best available package versions in transactions."
msgstr ""

#: ../dnf/cli/option_parser.py:175
msgid "run entirely from system cache, don't update cache"
msgstr "キャッシュから完全に実行し、キャッシュを更新しません"

#: ../dnf/cli/option_parser.py:179
msgid "config file location"
msgstr "構成ファイルの場所"

#: ../dnf/cli/option_parser.py:182
msgid "maximum command wait time"
msgstr "コマンドの最大待ち時間"

#: ../dnf/cli/option_parser.py:185
msgid "debugging output level"
msgstr "デバッグ情報の出力レベル"

#: ../dnf/cli/option_parser.py:188
msgid "dumps package metadata into files"
msgstr ""

#: ../dnf/cli/option_parser.py:191
msgid "dumps detailed solving results into files"
msgstr ""

#: ../dnf/cli/option_parser.py:194
msgid "show duplicates, in repos, in list/search commands"
msgstr "一覧/検索コマンドのリポジトリーの重複の表示"

#: ../dnf/cli/option_parser.py:197
msgid "error output level"
msgstr "エラー出力レベル"

#: ../dnf/cli/option_parser.py:199
msgid "debugging output level for rpm"
msgstr "rpm のデバッグ情報の出力レベル"

#: ../dnf/cli/option_parser.py:202
msgid "quiet operation"
msgstr "静かに処理をする"

#: ../dnf/cli/option_parser.py:204
msgid "verbose operation"
msgstr "冗長に処理をする"

#: ../dnf/cli/option_parser.py:206
msgid "answer yes for all questions"
msgstr "すべての問い合わせに「yes」で答える"

#: ../dnf/cli/option_parser.py:208
msgid "answer no for all questions"
msgstr ""

#: ../dnf/cli/option_parser.py:210
msgid "show Yum version and exit"
msgstr "Yum のバージョンを表示して終了する"

#: ../dnf/cli/option_parser.py:211
msgid "set install root"
msgstr "インストールのベース ディレクトリーを設定する"

#: ../dnf/cli/option_parser.py:220
msgid "exclude packages by name or glob"
msgstr ""

#: ../dnf/cli/option_parser.py:223
msgid "disable excludes"
msgstr ""

#: ../dnf/cli/option_parser.py:226
msgid "enable obsoletes processing during upgrades"
msgstr ""

#: ../dnf/cli/option_parser.py:228
msgid "disable all plugins"
msgstr ""

#: ../dnf/cli/option_parser.py:230
msgid "disable gpg signature checking"
msgstr ""

#: ../dnf/cli/option_parser.py:233
msgid "disable plugins by name"
msgstr "名前でプラグインを無効にする"

#: ../dnf/cli/option_parser.py:236
msgid "control whether color is used"
msgstr ""

#: ../dnf/cli/option_parser.py:238
msgid "override the value of $releasever in config and repo files"
msgstr ""

#: ../dnf/cli/option_parser.py:242
msgid "set arbitrary config and repo options"
msgstr "全体設定とリポジトリー オプションの任意に設定する"

#: ../dnf/cli/option_parser.py:270
msgid "List of Main Commands"
msgstr ""

#: ../dnf/cli/option_parser.py:271
msgid "List of Plugin Commands"
msgstr ""

#: ../dnf/cli/utils.py:108
msgid "Running"
msgstr "実行中"

#: ../dnf/cli/utils.py:109
msgid "Sleeping"
msgstr "スリープ中"

#: ../dnf/cli/utils.py:110
msgid "Uninterruptible"
msgstr "割り込み不可"

#: ../dnf/cli/utils.py:111
msgid "Zombie"
msgstr "ゾンビ"

#: ../dnf/cli/utils.py:112
msgid "Traced/Stopped"
msgstr "トレース/停止"

#: ../dnf/cli/utils.py:113
msgid "Unknown"
msgstr "不明"

#: ../dnf/cli/utils.py:133
#, python-format
msgid "  The application with PID %d is: %s"
msgstr ""

#: ../dnf/cli/utils.py:140
#, python-format
msgid "    Memory : %5s RSS (%5sB VSZ)"
msgstr "   メモリー: %5s RSS (%5sB VSZ)"

#: ../dnf/cli/utils.py:145
#, python-format
msgid "    Started: %s - %s ago"
msgstr "    開始   : %s - %s 秒経過"

#: ../dnf/cli/utils.py:147
#, python-format
msgid "    State  : %s"
msgstr ""

#: ../dnf/cli/cli.py:121
#, python-format
msgid "  Installed: %s-%s at %s"
msgstr "インストール: %s-%s (日時: %s)"

#: ../dnf/cli/cli.py:123
#, python-format
msgid "  Built    : %s at %s"
msgstr "  構築      : %s  (日時: %s)"

#: ../dnf/cli/cli.py:157
msgid "Disk Requirements:\n"
msgstr "ディスク要求:\n"

#: ../dnf/cli/cli.py:164
msgid ""
"Error Summary\n"
"-------------\n"
msgstr "エラーの要約\n-------------\n"

#: ../dnf/cli/cli.py:178
msgid "Trying to run the transaction but nothing to do. Exiting."
msgstr "トランザクションの実行を試みましたが、何もありませんでした。終了します。"

#: ../dnf/cli/cli.py:212
msgid "Exiting on user Command"
msgstr "ユーザーコマンドを終了しています"

#: ../dnf/cli/cli.py:217
msgid "Downloading Packages:"
msgstr "パッケージをダウンロードしています:"

#: ../dnf/cli/cli.py:223
#, python-format
msgid ""
"Error downloading packages:\n"
"%s"
msgstr ""

#: ../dnf/cli/cli.py:256
msgid ""
"Refusing to automatically import keys when running unattended.\n"
"Use \"-y\" to override."
msgstr ""

#: ../dnf/cli/cli.py:296 ../dnf/cli/cli.py:449 ../dnf/cli/cli.py:455
msgid "Obsoleting Packages"
msgstr "パッケージを不要にしています"

#: ../dnf/cli/cli.py:311 ../dnf/cli/commands/group.py:386
#: ../dnf/cli/commands/upgrade.py:88 ../dnf/cli/commands/__init__.py:1227
msgid "No packages marked for upgrade."
msgstr ""

#: ../dnf/cli/cli.py:333
msgid "No packages marked for distribution synchronization."
msgstr ""

#: ../dnf/cli/cli.py:364 ../dnf/cli/cli.py:759 ../dnf/cli/cli.py:788
#: ../dnf/cli/commands/install.py:99 ../dnf/cli/commands/__init__.py:855
#, python-format
msgid "No package %s%s%s available."
msgstr "パッケージ %s%s%s は利用できません。"

#: ../dnf/cli/cli.py:369
#, python-format
msgid "No match for available package: %s"
msgstr ""

#: ../dnf/cli/cli.py:374 ../dnf/cli/main.py:159
#: ../dnf/cli/commands/install.py:107 ../dnf/cli/commands/reinstall.py:99
#: ../dnf/cli/commands/__init__.py:863 ../dnf/cli/commands/__init__.py:953
#: ../dnf/cli/commands/__init__.py:1020 ../dnf/cli/commands/__init__.py:1072
#: ../dnf/cli/commands/__init__.py:1131
msgid "Nothing to do."
msgstr ""

#: ../dnf/cli/cli.py:425
msgid "Installed Packages"
msgstr "インストール済みパッケージ"

#: ../dnf/cli/cli.py:433
msgid "Available Packages"
msgstr "利用可能なパッケージ"

#: ../dnf/cli/cli.py:437
msgid "Extra Packages"
msgstr "外部パッケージ"

#: ../dnf/cli/cli.py:441
msgid "Upgraded Packages"
msgstr ""

#: ../dnf/cli/cli.py:457
msgid "Recently Added Packages"
msgstr "最近追加したパッケージ"

#: ../dnf/cli/cli.py:461
msgid "No matching Packages to list"
msgstr "表示するパッケージはありません"

#: ../dnf/cli/cli.py:571 ../dnf/cli/cli.py:1229
msgid "No Matches found"
msgstr "見つかりませんでした"

#: ../dnf/cli/cli.py:600
msgid "Cleaning repos: "
msgstr "リポジトリーを清掃しています: "

#: ../dnf/cli/cli.py:604
msgid "Cleaning up Everything"
msgstr "すべて掃除しています"

#: ../dnf/cli/cli.py:618
msgid "Cleaning up Packages"
msgstr "パッケージを掃除しています"

#: ../dnf/cli/cli.py:621
msgid "Cleaning up xml metadata"
msgstr "XML メタデータを掃除しています"

#: ../dnf/cli/cli.py:624
msgid "Cleaning up database cache"
msgstr "データベースキャッシュを掃除しています"

#: ../dnf/cli/cli.py:627
msgid "Cleaning up expire-cache metadata"
msgstr "期限切れのメタデータキャッシュを掃除しています"

#: ../dnf/cli/cli.py:630
msgid "Cleaning up cached rpmdb data"
msgstr "キャッシュ済み rpmdb データを掃除しています"

#: ../dnf/cli/cli.py:679 ../dnf/cli/commands/__init__.py:1468
#: ../dnf/cli/commands/__init__.py:1484
msgid "No transaction ID given"
msgstr "指定されたトランザクション ID がありません。"

#: ../dnf/cli/cli.py:688 ../dnf/cli/cli.py:696 ../dnf/cli/output.py:2107
msgid "Bad transaction ID given"
msgstr "指定されたトランザクション ID がおかしいです"

#: ../dnf/cli/cli.py:704
msgid "Not found given transaction ID"
msgstr "指定されたトランザクション ID が見つかりません"

#: ../dnf/cli/cli.py:713 ../dnf/cli/commands/__init__.py:1470
#: ../dnf/cli/commands/__init__.py:1486
msgid "Found more than one transaction ID!"
msgstr "ひとつ以上のトランザクション ID が見つかりません!"

#: ../dnf/cli/cli.py:730
#, python-format
msgid "Transaction history is incomplete, before %u."
msgstr ""

#: ../dnf/cli/cli.py:732
#, python-format
msgid "Transaction history is incomplete, after %u."
msgstr ""

#: ../dnf/cli/cli.py:755 ../dnf/cli/cli.py:784
#, python-format
msgid "No package %s%s%s installed."
msgstr ""

#: ../dnf/cli/cli.py:842
#, python-format
msgid "Unknown repo: '%s'"
msgstr ""

#: ../dnf/cli/cli.py:913
msgid "You need to give some command"
msgstr "いくつかのコマンドを指定する必要があります"

#: ../dnf/cli/cli.py:920
#, python-format
msgid "No such command: %s. Please use %s --help"
msgstr "コマンド「%s」が見つかりません。「%s --help」を実行してください。"

#: ../dnf/cli/cli.py:1023 ../dnf/cli/cli.py:1027
#, python-format
msgid "Config error: %s"
msgstr ""

#: ../dnf/cli/cli.py:1142
#, python-format
msgid "Command \"%s\" already defined"
msgstr "コマンド「%s」はすでに定義済みです"

#: ../dnf/cli/cli.py:1207
#, python-format
msgid "N/S Matched: %s"
msgstr ""

#: ../dnf/cli/cli.py:1210
#, python-format
msgid "Matched: %s"
msgstr "一致: %s"

#: ../dnf/cli/cli.py:1228
#, python-format
msgid "Warning: No matches found for: %s"
msgstr "警告: 一致するものが見つかりません: %s"

#: ../dnf/cli/cli.py:1232
msgid "Writing out repository metadata for debugging."
msgstr ""

#: ../dnf/cli/main.py:70 ../dnf/cli/main.py:150
#, python-format
msgid "Error: %s"
msgstr "エラー: %s"

#: ../dnf/cli/main.py:87
msgid "Terminated."
msgstr ""

#: ../dnf/cli/main.py:119
msgid "No read/execute access in current directory, moving to /"
msgstr ""

#: ../dnf/cli/main.py:146
msgid "Resolving dependencies"
msgstr ""

#: ../dnf/cli/main.py:153
msgid "Dependencies resolved."
msgstr ""

#: ../dnf/cli/main.py:186
msgid "Complete!"
msgstr "完了しました!"

#: ../dnf/cli/output.py:775
#, python-format
msgid "Name        : %s%s%s"
msgstr "名前                : %s%s%s"

#: ../dnf/cli/output.py:776
#, python-format
msgid "Arch        : %s"
msgstr "アーキテクチャ      : %s"

#: ../dnf/cli/output.py:778
#, python-format
msgid "Epoch       : %s"
msgstr "エポック            : %s"

#: ../dnf/cli/output.py:779
#, python-format
msgid "Version     : %s"
msgstr "バージョン          : %s"

#: ../dnf/cli/output.py:780
#, python-format
msgid "Release     : %s"
msgstr "リリース            : %s"

#: ../dnf/cli/output.py:781
#, python-format
msgid "Size        : %s"
msgstr "容量                : %s"

#: ../dnf/cli/output.py:782 ../dnf/cli/output.py:1101
#, python-format
msgid "Repo        : %s"
msgstr "リポジトリー        : %s"

#: ../dnf/cli/output.py:784
#, python-format
msgid "From repo   : %s"
msgstr "提供元リポジトリー  : %s"

#: ../dnf/cli/output.py:789
#, python-format
msgid "Buildtime   : %s"
msgstr "ビルド日時          : %s"

#: ../dnf/cli/output.py:791
#, python-format
msgid "Install time: %s"
msgstr "インストール日時    : %s "

#: ../dnf/cli/output.py:799
#, python-format
msgid "Installed by: %s"
msgstr "インストール済み容量: %s"

#: ../dnf/cli/output.py:806
#, python-format
msgid "Changed by  : %s"
msgstr "変更者              : %s"

#: ../dnf/cli/output.py:807
msgid "Summary     : "
msgstr "要約                : "

#: ../dnf/cli/output.py:809 ../dnf/cli/output.py:1116
#, python-format
msgid "URL         : %s"
msgstr "URL                 : %s"

#: ../dnf/cli/output.py:810
msgid "License     : "
msgstr "ライセンス          : "

#: ../dnf/cli/output.py:811 ../dnf/cli/output.py:1113
msgid "Description : "
msgstr "説明                : "

#: ../dnf/cli/output.py:929
msgid "y"
msgstr "y"

#: ../dnf/cli/output.py:929
msgid "yes"
msgstr "はい"

#: ../dnf/cli/output.py:930
msgid "n"
msgstr "n"

#: ../dnf/cli/output.py:930
msgid "no"
msgstr "いいえ"

#: ../dnf/cli/output.py:933
msgid "Is this ok [y/N]: "
msgstr "これでいいですか? [y/N]"

#: ../dnf/cli/output.py:936
msgid "Is this ok [Y/n]: "
msgstr ""

#: ../dnf/cli/output.py:1013
#, python-format
msgid ""
"\n"
"Group: %s"
msgstr "\nグループ: %s"

#: ../dnf/cli/output.py:1017
#, python-format
msgid " Group-Id: %s"
msgstr " グループ ID: %s"

#: ../dnf/cli/output.py:1019
#, python-format
msgid " Description: %s"
msgstr " 説明: %s"

#: ../dnf/cli/output.py:1021
#, python-format
msgid " Language: %s"
msgstr " 言語: %s"

#: ../dnf/cli/output.py:1024
msgid " Mandatory Packages:"
msgstr " 強制的なパッケージ:"

#: ../dnf/cli/output.py:1025
msgid " Default Packages:"
msgstr " 標準パッケージ:"

#: ../dnf/cli/output.py:1026
msgid " Optional Packages:"
msgstr " オプション パッケージ:"

#: ../dnf/cli/output.py:1027
msgid " Conditional Packages:"
msgstr " 条件付パッケージ:"

#: ../dnf/cli/output.py:1053
#, python-format
msgid "package: %s"
msgstr "パッケージ    : %s"

#: ../dnf/cli/output.py:1055
msgid "  No dependencies for this package"
msgstr "  このパッケージの依存はありません"

#: ../dnf/cli/output.py:1060
#, python-format
msgid "  dependency: %s"
msgstr "  依存性      : %s"

#: ../dnf/cli/output.py:1062
msgid "   Unsatisfied dependency"
msgstr "  満たされていない依存性"

#: ../dnf/cli/output.py:1108
msgid "Matched from:"
msgstr "一致          :"

#: ../dnf/cli/output.py:1119
#, python-format
msgid "License     : %s"
msgstr "ライセンス    : %s"

#: ../dnf/cli/output.py:1122
#, python-format
msgid "Filename    : %s"
msgstr "ファイル名    : %s"

#: ../dnf/cli/output.py:1126
msgid "Other       : "
msgstr "その他        : "

#: ../dnf/cli/output.py:1182
msgid "There was an error calculating total download size"
msgstr "総ダウンロード容量の計算中にエラーです。"

#: ../dnf/cli/output.py:1187
#, python-format
msgid "Total size: %s"
msgstr "合計容量: %s"

#: ../dnf/cli/output.py:1190
#, python-format
msgid "Total download size: %s"
msgstr "総ダウンロード容量: %s"

#: ../dnf/cli/output.py:1194 ../dnf/cli/output.py:1217
#, python-format
msgid "Installed size: %s"
msgstr "インストール済み容量: %s"

#: ../dnf/cli/output.py:1213
msgid "There was an error calculating installed size"
msgstr "インストール容量の計算中にエラーです。"

#: ../dnf/cli/output.py:1257
msgid "Removing"
msgstr "削除"

#: ../dnf/cli/output.py:1281 ../dnf/cli/output.py:2162
msgid "Package"
msgstr ""

#: ../dnf/cli/output.py:1281
msgid "Arch"
msgstr "アーキテクチャ"

#: ../dnf/cli/output.py:1282
msgid "Version"
msgstr "バージョン"

#: ../dnf/cli/output.py:1282
msgid "Repository"
msgstr "リポジトリー"

#: ../dnf/cli/output.py:1283
msgid "Size"
msgstr "容量"

#: ../dnf/cli/output.py:1295
#, python-format
msgid "     replacing  %s%s%s.%s %s\n"
msgstr "     置き換えています  %s%s%s.%s %s\n"

#: ../dnf/cli/output.py:1303
#, python-format
msgid ""
"\n"
"Transaction Summary\n"
"%s\n"
msgstr "\nトランザクションの要約\n%s\n"

#: ../dnf/cli/output.py:1308 ../dnf/cli/output.py:1957
#: ../dnf/cli/output.py:1958
msgid "Install"
msgstr "インストール"

#: ../dnf/cli/output.py:1309 ../dnf/cli/output.py:1966
msgid "Upgrade"
msgstr ""

#: ../dnf/cli/output.py:1310
msgid "Remove"
msgstr ""

#: ../dnf/cli/output.py:1311 ../dnf/cli/output.py:1964
msgid "Downgrade"
msgstr "ダウングレード"

#: ../dnf/cli/output.py:1397
msgid "Reinstalled"
msgstr ""

#: ../dnf/cli/output.py:1398
msgid "Removed"
msgstr "削除しました"

#: ../dnf/cli/output.py:1399 ../dnf/cli/output.py:1802
#: ../dnf/cli/output.py:1804
msgid "Installed"
msgstr "インストール"

#: ../dnf/cli/output.py:1400 ../dnf/cli/output.py:1803
#: ../dnf/cli/output.py:1967
msgid "Upgraded"
msgstr ""

#: ../dnf/cli/output.py:1401 ../dnf/cli/output.py:1803
#: ../dnf/cli/output.py:1965
msgid "Downgraded"
msgstr "ダウングレード済み"

#: ../dnf/cli/output.py:1457
msgid "Total"
msgstr "合計"

#: ../dnf/cli/output.py:1478
msgid "I"
msgstr "I"

#: ../dnf/cli/output.py:1479
msgid "O"
msgstr "O"

#: ../dnf/cli/output.py:1480
msgid "E"
msgstr "E"

#: ../dnf/cli/output.py:1481
msgid "R"
msgstr "R"

#: ../dnf/cli/output.py:1482
msgid "D"
msgstr "D"

#: ../dnf/cli/output.py:1483
msgid "U"
msgstr "U"

#: ../dnf/cli/output.py:1497
msgid "<unset>"
msgstr "<未設定>"

#: ../dnf/cli/output.py:1498
msgid "System"
msgstr "システム"

#: ../dnf/cli/output.py:1567
#, python-format
msgid "Skipping merged transaction %d to %d, as it overlaps"
msgstr ""

#: ../dnf/cli/output.py:1577 ../dnf/cli/output.py:1708
msgid "No transactions"
msgstr "トランザクションがありません"

#: ../dnf/cli/output.py:1602 ../dnf/cli/output.py:2151
#: ../dnf/cli/output.py:2230
msgid "Bad transaction IDs, or package(s), given"
msgstr "指定されたトランザクション ID、またはパッケージがおかしいです"

#: ../dnf/cli/output.py:1649
msgid "Command line"
msgstr "コマンドライン"

#: ../dnf/cli/output.py:1651 ../dnf/cli/output.py:2039
msgid "Login user"
msgstr "ログイン ユーザー"

#: ../dnf/cli/output.py:1652 ../dnf/cli/output.py:2160
msgid "ID"
msgstr "ID"

#: ../dnf/cli/output.py:1654
msgid "Date and time"
msgstr "日時"

#: ../dnf/cli/output.py:1655 ../dnf/cli/output.py:2041
#: ../dnf/cli/output.py:2161
msgid "Action(s)"
msgstr "操作"

#: ../dnf/cli/output.py:1656 ../dnf/cli/output.py:2042
msgid "Altered"
msgstr "変更"

#: ../dnf/cli/output.py:1741 ../dnf/cli/output.py:2115
msgid "No transaction ID, or package, given"
msgstr "指定されたトランザクション ID、またはパッケージが見つかりません。"

#: ../dnf/cli/output.py:1802
msgid "Erased"
msgstr "削除"

#: ../dnf/cli/output.py:1804
msgid "Not installed"
msgstr "未インストール"

#: ../dnf/cli/output.py:1805
msgid "Older"
msgstr "より古い"

#: ../dnf/cli/output.py:1805
msgid "Newer"
msgstr "より新しい"

#: ../dnf/cli/output.py:1842 ../dnf/cli/output.py:1844
#: ../dnf/cli/output.py:2251
msgid "Transaction ID :"
msgstr "トランザクション ID :"

#: ../dnf/cli/output.py:1846 ../dnf/cli/output.py:2253
msgid "Begin time     :"
msgstr "開始時間            :"

#: ../dnf/cli/output.py:1849 ../dnf/cli/output.py:1851
msgid "Begin rpmdb    :"
msgstr "開始 rpmdb          :"

#: ../dnf/cli/output.py:1867
#, python-format
msgid "(%u seconds)"
msgstr "(%u 秒)"

#: ../dnf/cli/output.py:1869
#, python-format
msgid "(%u minutes)"
msgstr "(%u 分)"

#: ../dnf/cli/output.py:1871
#, python-format
msgid "(%u hours)"
msgstr "(%u 時間)"

#: ../dnf/cli/output.py:1873
#, python-format
msgid "(%u days)"
msgstr "(%u 日)"

#: ../dnf/cli/output.py:1874
msgid "End time       :"
msgstr "終了時間            :"

#: ../dnf/cli/output.py:1877 ../dnf/cli/output.py:1879
msgid "End rpmdb      :"
msgstr "終了 rpmdb          :"

#: ../dnf/cli/output.py:1882 ../dnf/cli/output.py:1884
msgid "User           :"
msgstr "ユーザー            :"

#: ../dnf/cli/output.py:1888 ../dnf/cli/output.py:1891
#: ../dnf/cli/output.py:1893 ../dnf/cli/output.py:1895
#: ../dnf/cli/output.py:1897
msgid "Return-Code    :"
msgstr "終了コード          :"

#: ../dnf/cli/output.py:1888 ../dnf/cli/output.py:1893
msgid "Aborted"
msgstr "中断しました"

#: ../dnf/cli/output.py:1891
msgid "Failures:"
msgstr "失敗:"

#: ../dnf/cli/output.py:1895
msgid "Failure:"
msgstr "失敗しました:"

#: ../dnf/cli/output.py:1897
msgid "Success"
msgstr "成功"

#: ../dnf/cli/output.py:1902 ../dnf/cli/output.py:1904
#: ../dnf/cli/output.py:2282
msgid "Command Line   :"
msgstr "コマンドライン      :"

#: ../dnf/cli/output.py:1913
#, python-format
msgid "Additional non-default information stored: %d"
msgstr "保存済みの追加の非標準な情報: %d"

#: ../dnf/cli/output.py:1918
msgid "Transaction performed with:"
msgstr "トランザクションの実行:"

#: ../dnf/cli/output.py:1922
msgid "Packages Altered:"
msgstr "切り替えたパッケージ:"

#: ../dnf/cli/output.py:1926
msgid "Packages Skipped:"
msgstr "パッケージを飛ばします:"

#: ../dnf/cli/output.py:1934
msgid "Rpmdb Problems:"
msgstr "Rpmdb の問題:"

#: ../dnf/cli/output.py:1945
msgid "Scriptlet output:"
msgstr "スクリプトの出力:"

#: ../dnf/cli/output.py:1951
msgid "Errors:"
msgstr "エラー:"

#: ../dnf/cli/output.py:1959
msgid "Dep-Install"
msgstr "依存インストール"

#: ../dnf/cli/output.py:1960
msgid "Obsoleted"
msgstr "不要"

#: ../dnf/cli/output.py:1962
msgid "Erase"
msgstr "削除"

#: ../dnf/cli/output.py:1963
msgid "Reinstall"
msgstr "再インストール"

#: ../dnf/cli/output.py:2040
msgid "Time"
msgstr "時間"

#: ../dnf/cli/output.py:2066
msgid "Last day"
msgstr "昨日"

#: ../dnf/cli/output.py:2067
msgid "Last week"
msgstr "先週"

#: ../dnf/cli/output.py:2068
msgid "Last 2 weeks"
msgstr "最近の 2 週間"

#: ../dnf/cli/output.py:2069
msgid "Last 3 months"
msgstr "最近の 3 ヶ月間"

#: ../dnf/cli/output.py:2070
msgid "Last 6 months"
msgstr "最近の 6 ヶ月間"

#: ../dnf/cli/output.py:2071
msgid "Last year"
msgstr "昨年"

#: ../dnf/cli/output.py:2072
msgid "Over a year ago"
msgstr "1 年以上前"

#: ../dnf/cli/output.py:2119
#, python-format
msgid "No Transaction %s found"
msgstr "トランザクション %s が見つかりません。"

#: ../dnf/cli/output.py:2125
msgid "Transaction ID:"
msgstr "トランザクション ID:"

#: ../dnf/cli/output.py:2126
msgid "Available additional history information:"
msgstr "追加の履歴情報が利用できます:"

#: ../dnf/cli/output.py:2139
#, python-format
msgid "%s: No additional data found by this name"
msgstr "%s: この名前から追加のデータが見つかりません。"

#: ../dnf/cli/output.py:2254
msgid "Package        :"
msgstr ""

#: ../dnf/cli/output.py:2255
msgid "State          :"
msgstr ""

#: ../dnf/cli/output.py:2258
msgid "Size           :"
msgstr ""

#: ../dnf/cli/output.py:2260
msgid "Build host     :"
msgstr ""

#: ../dnf/cli/output.py:2263
msgid "Build time     :"
msgstr ""

#: ../dnf/cli/output.py:2265
msgid "Packager       :"
msgstr ""

#: ../dnf/cli/output.py:2267
msgid "Vendor         :"
msgstr ""

#: ../dnf/cli/output.py:2269
msgid "License        :"
msgstr ""

#: ../dnf/cli/output.py:2271
msgid "URL            :"
msgstr ""

#: ../dnf/cli/output.py:2273
msgid "Source RPM     :"
msgstr ""

#: ../dnf/cli/output.py:2276
msgid "Commit Time    :"
msgstr ""

#: ../dnf/cli/output.py:2278
msgid "Committer      :"
msgstr ""

#: ../dnf/cli/output.py:2280
msgid "Reason         :"
msgstr ""

#: ../dnf/cli/output.py:2284
msgid "From repo      :"
msgstr ""

#: ../dnf/cli/output.py:2288
msgid "Installed by   :"
msgstr ""

#: ../dnf/cli/output.py:2292
msgid "Changed by     :"
msgstr ""

#: ../dnf/cli/output.py:2324
msgid "installed"
msgstr "インストール"

#: ../dnf/cli/output.py:2325
msgid "an upgrade"
msgstr ""

#: ../dnf/cli/output.py:2326
msgid "erased"
msgstr "削除"

#: ../dnf/cli/output.py:2327
msgid "reinstalled"
msgstr "再インストール"

#: ../dnf/cli/output.py:2328
msgid "a downgrade"
msgstr ""

#: ../dnf/cli/output.py:2329
msgid "obsoleting"
msgstr ""

#: ../dnf/cli/output.py:2330
msgid "upgraded"
msgstr ""

#: ../dnf/cli/output.py:2331
msgid "obsoleted"
msgstr "不要"

#: ../dnf/cli/output.py:2332
msgid "downgraded"
msgstr ""

#: ../dnf/cli/output.py:2336
#, python-format
msgid "---> Package %s.%s %s will be %s"
msgstr ""

#: ../dnf/cli/output.py:2344
msgid "--> Starting dependency resolution"
msgstr ""

#: ../dnf/cli/output.py:2349
msgid "--> Running transaction check"
msgstr "--> トランザクションの確認を実行しています。"

#: ../dnf/cli/output.py:2355
msgid "--> Restarting Dependency Resolution with new changes."
msgstr "--> 新しい変更と依存性の解決を再開しています。"

#: ../dnf/cli/output.py:2361
msgid "--> Finished dependency resolution"
msgstr ""

#: ../dnf/cli/output.py:2373 ../dnf/cli/output.py:2386
#, python-format
msgid "--> Processing Dependency: %s for package: %s"
msgstr "--> 依存性の処理をしています: %s のパッケージ: %s"

#: ../dnf/cli/output.py:2398
#, python-format
msgid "---> Keeping package: %s"
msgstr "---> 維持しています: %s"

#: ../dnf/cli/output.py:2405
#, python-format
msgid "--> Unresolved Dependency: %s"
msgstr "--> 未解決の依存性: %s"

#: ../dnf/cli/output.py:2418 ../dnf/cli/output.py:2431
#, python-format
msgid "--> Processing Conflict: %s conflicts %s"
msgstr "--> 衝突を処理しています: %s は %s と衝突しています"

#: ../dnf/cli/output.py:2437
msgid "--> Populating transaction set with selected packages. Please wait."
msgstr ""

#: ../dnf/cli/output.py:2573
msgid "Verifying"
msgstr ""

#: ../dnf/cli/commands/install.py:38
msgid "Install a package or packages on your system"
msgstr "システムにパッケージをインストールする"

#: ../dnf/cli/commands/install.py:39 ../dnf/cli/commands/reinstall.py:39
#: ../dnf/cli/commands/upgrade.py:38 ../dnf/cli/commands/distrosync.py:35
#: ../dnf/cli/commands/downgrade.py:35 ../dnf/cli/commands/__init__.py:247
#: ../dnf/cli/commands/__init__.py:271 ../dnf/cli/commands/__init__.py:317
#: ../dnf/cli/commands/__init__.py:469
msgid "PACKAGE"
msgstr ""

#: ../dnf/cli/commands/install.py:83
#, python-format
msgid "Warning: Group '%s' does not exist."
msgstr ""

#: ../dnf/cli/commands/install.py:88
msgid "No packages in any requested group available to install or upgrade."
msgstr ""

#: ../dnf/cli/commands/reinstall.py:38
msgid "reinstall a package"
msgstr "パッケージの再インストール"

#: ../dnf/cli/commands/reinstall.py:81 ../dnf/cli/commands/upgrade.py:82
#: ../dnf/cli/commands/__init__.py:345 ../dnf/cli/commands/__init__.py:935
#: ../dnf/cli/commands/__init__.py:1002 ../dnf/cli/commands/__init__.py:1123
#: ../dnf/cli/commands/__init__.py:1175 ../dnf/cli/commands/__init__.py:1221
#, python-format
msgid "No match for argument: %s"
msgstr ""

#: ../dnf/cli/commands/reinstall.py:88 ../dnf/cli/commands/__init__.py:942
#: ../dnf/cli/commands/__init__.py:1009
#, python-format
msgid " (from %s)"
msgstr " (%s から)"

#: ../dnf/cli/commands/reinstall.py:89 ../dnf/cli/commands/__init__.py:943
#: ../dnf/cli/commands/__init__.py:1010
#, python-format
msgid "Installed package %s%s%s%s not available."
msgstr "インストール済みパッケージ %s%s%s%s は利用できません。"

#: ../dnf/cli/commands/group.py:45
msgid "Error: Need a group or list of groups"
msgstr "エラー: グループ化グループの一覧が必要です"

#: ../dnf/cli/commands/group.py:89
#, python-format
msgid "No relevant match for the specified '%s'."
msgstr ""

#: ../dnf/cli/commands/group.py:105
msgid "Display, or use, the groups information"
msgstr ""

#: ../dnf/cli/commands/group.py:106
msgid "GROUP"
msgstr ""

#: ../dnf/cli/commands/group.py:138
msgid "No group data available for configured repositories."
msgstr ""

#: ../dnf/cli/commands/group.py:185
msgid "No Groups Available in any repository"
msgstr "いずれかのリポジトリーに利用できるグループはありません"

#: ../dnf/cli/commands/group.py:198
#, python-format
msgid "Warning: Group %s does not exist."
msgstr "警告: グループ %s が存在しません。"

#: ../dnf/cli/commands/group.py:214
msgid "No packages in any requested groups available to install."
msgstr ""

#: ../dnf/cli/commands/group.py:231
#, python-format
msgid "Warning: No groups match: %s"
msgstr "警告: 一致するグループがありません: %s"

#: ../dnf/cli/commands/group.py:254
msgid "Available environment groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:255
msgid "Installed environment groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:260
msgid "Installed groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:266
msgid "Installed language groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:272
msgid "Available groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:278
msgid "Available language groups:"
msgstr ""

#: ../dnf/cli/commands/group.py:293
#, python-format
msgid "Environments marked installed: %s"
msgstr ""

#: ../dnf/cli/commands/group.py:298
#, python-format
msgid "Groups marked installed: %s"
msgstr ""

#: ../dnf/cli/commands/group.py:310
#, python-format
msgid "Environments marked removed: %s"
msgstr ""

#: ../dnf/cli/commands/group.py:315
#, python-format
msgid "Groups marked removed: %s"
msgstr ""

#: ../dnf/cli/commands/group.py:335
msgid "No packages to remove from given groups."
msgstr ""

#: ../dnf/cli/commands/group.py:356
msgid "Installed Groups:"
msgstr "インストール済みグループ:"

#: ../dnf/cli/commands/group.py:362
msgid "Installed Language Groups:"
msgstr "インストール済みの言語グループ:"

#: ../dnf/cli/commands/group.py:368
msgid "Available Groups:"
msgstr "利用可能なグループ"

#: ../dnf/cli/commands/group.py:374
msgid "Available Language Groups:"
msgstr "利用できる言語グループ:"

#: ../dnf/cli/commands/group.py:417
#, python-format
msgid "Invalid groups sub-command, use: %s."
msgstr ""

#: ../dnf/cli/commands/upgrade.py:37
msgid "Upgrade a package or packages on your system"
msgstr ""

#: ../dnf/cli/commands/distrosync.py:34
msgid "Synchronize installed packages to the latest available versions"
msgstr "最新の利用可能なバージョンへインストール済みパッケージを同期する"

#: ../dnf/cli/commands/downgrade.py:34
msgid "downgrade a package"
msgstr "パッケージのダウングレード"

#: ../dnf/cli/commands/__init__.py:44
#, python-format
msgid "To diagnose the problem, try running: '%s'."
msgstr ""

#: ../dnf/cli/commands/__init__.py:46
#, python-format
msgid "To fix inconsistent RPMDB, try running: '%s'."
msgstr ""

#: ../dnf/cli/commands/__init__.py:48
#, python-format
msgid "If the above doesn't help please report this error at '%s'."
msgstr ""

#: ../dnf/cli/commands/__init__.py:56
msgid " Mini usage:\n"
msgstr ""

#: ../dnf/cli/commands/__init__.py:71
msgid ""
"\n"
"You have enabled checking of packages via GPG keys. This is a good thing.\n"
"However, you do not have any GPG public keys installed. You need to download\n"
"the keys for packages you wish to install and install them.\n"
"You can do that by running the command:\n"
"    rpm --import public.gpg.key\n"
"\n"
"\n"
"Alternatively you can specify the url to the key you would like to use\n"
"for a repository in the 'gpgkey' option in a repository section and yum\n"
"will install it for you.\n"
"\n"
"For more information contact your distribution or package provider.\n"
msgstr ""

#: ../dnf/cli/commands/__init__.py:86
#, python-format
msgid "Problem repository: %s"
msgstr "問題のリポジトリー: %s"

#: ../dnf/cli/commands/__init__.py:100
#, python-format
msgid "Error: Need to pass a list of pkgs to %s"
msgstr "エラー: パッケージの一覧を %s に渡す必要があります"

#: ../dnf/cli/commands/__init__.py:116
msgid "Error: Need an item to match"
msgstr "エラー: 一致する項目が必要です"

#: ../dnf/cli/commands/__init__.py:133
#, python-format
msgid "Error: clean requires an option: %s"
msgstr "エラー: clean は引数をひとつ要求します: %s"

#: ../dnf/cli/commands/__init__.py:138
#, python-format
msgid "Error: invalid clean argument: %r"
msgstr "エラー: 不正な clean の引数です: %r"

#: ../dnf/cli/commands/__init__.py:157
msgid ""
"There are no enabled repos.\n"
" Run \"yum repolist all\" to see the repos you have.\n"
" You can enable repos with yum-config-manager --enable <repo>"
msgstr "有効なリポジトリーがありません。\n 「yum repolist all」を実行し、所持するリポジトリーを参照してください。\n 「yum-config-manager --enable <repo>」でリポジトリーを有効にできます。"

#: ../dnf/cli/commands/__init__.py:246
msgid "Upgrade a package on your system to the specified version"
msgstr ""

#: ../dnf/cli/commands/__init__.py:270
msgid "Display details about a package or group of packages"
msgstr "パッケージもしくはパッケージのグループについての詳細を表示する"

#: ../dnf/cli/commands/__init__.py:300
msgid "List a package or groups of packages"
msgstr "パッケージグループの一覧を表示する"

#: ../dnf/cli/commands/__init__.py:316
msgid "Remove a package or packages from your system"
msgstr "システムから削除するパッケージ"

#: ../dnf/cli/commands/__init__.py:351 ../dnf/cli/commands/__init__.py:1181
msgid "No packages marked for removal."
msgstr ""

#: ../dnf/cli/commands/__init__.py:359
msgid "Generate the metadata cache"
msgstr "メタデータキャッシュを生成する"

#: ../dnf/cli/commands/__init__.py:372
msgid "Making cache files for all metadata files."
msgstr "すべてのメタデータファイルのキャッシュを作成します。"

#: ../dnf/cli/commands/__init__.py:379
msgid "Metadata timer caching disabled when running on a battery."
msgstr ""

#: ../dnf/cli/commands/__init__.py:384
msgid "Metadata timer caching disabled."
msgstr ""

#: ../dnf/cli/commands/__init__.py:389
msgid "Metadata cache refreshed recently."
msgstr ""

#: ../dnf/cli/commands/__init__.py:412
msgid "Metadata cache created."
msgstr ""

#: ../dnf/cli/commands/__init__.py:421
msgid "Remove cached data"
msgstr "キャッシュデータを削除する"

#: ../dnf/cli/commands/__init__.py:445
msgid "Find what package provides the given value"
msgstr "指定値を提供するパッケージを検索する"

#: ../dnf/cli/commands/__init__.py:446 ../dnf/cli/commands/__init__.py:502
msgid "SOME_STRING"
msgstr ""

#: ../dnf/cli/commands/__init__.py:468
msgid "Check for available package upgrades"
msgstr ""

#: ../dnf/cli/commands/__init__.py:501
msgid "Search package details for the given string"
msgstr "指定した文字列でパッケージの詳細を検索する"

#: ../dnf/cli/commands/__init__.py:514
msgid "Searching Packages: "
msgstr "パッケージの検索中: "

#: ../dnf/cli/commands/__init__.py:524
msgid "Display the configured software repositories"
msgstr "ソフトウェアリポジトリーの構成を表示する"

#: ../dnf/cli/commands/__init__.py:583 ../dnf/cli/commands/__init__.py:584
msgid "enabled"
msgstr "有効"

#: ../dnf/cli/commands/__init__.py:602 ../dnf/cli/commands/__init__.py:603
msgid "disabled"
msgstr "無効"

#: ../dnf/cli/commands/__init__.py:618
msgid "Repo-id      : "
msgstr "リポジトリー ID            : "

#: ../dnf/cli/commands/__init__.py:619
msgid "Repo-name    : "
msgstr "リポジトリーの名前         : "

#: ../dnf/cli/commands/__init__.py:622
msgid "Repo-status  : "
msgstr "リポジトリーの状態         : "

#: ../dnf/cli/commands/__init__.py:625
msgid "Repo-revision: "
msgstr "リポジトリーのリビジョン   : "

#: ../dnf/cli/commands/__init__.py:629
msgid "Repo-tags    : "
msgstr "リポジトリーのタグ         : "

#: ../dnf/cli/commands/__init__.py:634
msgid "Repo-distro-tags: "
msgstr ""

#: ../dnf/cli/commands/__init__.py:639
msgid "Repo-updated : "
msgstr "リポジトリー更新日         : "

#: ../dnf/cli/commands/__init__.py:641
msgid "Repo-pkgs    : "
msgstr "リポジトリー内パッケージ数 : "

#: ../dnf/cli/commands/__init__.py:642
msgid "Repo-size    : "
msgstr "リポジトリー容量           : "

#: ../dnf/cli/commands/__init__.py:645
msgid "Repo-metalink: "
msgstr "リポジトリーメタリンク     : "

#: ../dnf/cli/commands/__init__.py:649
msgid "  Updated    : "
msgstr "  更新日                   : "

#: ../dnf/cli/commands/__init__.py:652
msgid "Repo-mirrors : "
msgstr "リポジトリーのミラー       : "

#: ../dnf/cli/commands/__init__.py:656 ../dnf/cli/commands/__init__.py:660
msgid "Repo-baseurl : "
msgstr "リポジトリー基準 URL       : "

#: ../dnf/cli/commands/__init__.py:664
#, python-format
msgid "Never (last: %s)"
msgstr "ずっと (最終: %s)"

#: ../dnf/cli/commands/__init__.py:666
#, python-format
msgid "Instant (last: %s)"
msgstr "インスタント (最終: %s)"

#: ../dnf/cli/commands/__init__.py:669
#, python-format
msgid "%s second(s) (last: %s)"
msgstr "%s 秒 (最終: %s)"

#: ../dnf/cli/commands/__init__.py:671
msgid "Repo-expire  : "
msgstr "リポジトリーの期限         : "

#: ../dnf/cli/commands/__init__.py:674
msgid "Repo-exclude : "
msgstr "リポジトリーの除外         : "

#: ../dnf/cli/commands/__init__.py:678
msgid "Repo-include : "
msgstr "リポジトリーの内包         : "

#: ../dnf/cli/commands/__init__.py:682
msgid "Repo-excluded: "
msgstr "リポジトリーの除外数       : "

#: ../dnf/cli/commands/__init__.py:686
msgid "Repo-filename: "
msgstr ""

#: ../dnf/cli/commands/__init__.py:695 ../dnf/cli/commands/__init__.py:724
msgid "repo id"
msgstr "リポジトリー ID"

#: ../dnf/cli/commands/__init__.py:712 ../dnf/cli/commands/__init__.py:713
#: ../dnf/cli/commands/__init__.py:731
msgid "status"
msgstr "状態"

#: ../dnf/cli/commands/__init__.py:725
msgid "repo name"
msgstr "リポジトリー名"

#: ../dnf/cli/commands/__init__.py:846 ../dnf/cli/commands/__init__.py:924
#: ../dnf/cli/commands/__init__.py:991
msgid "No package available."
msgstr ""

#: ../dnf/cli/commands/__init__.py:922
msgid "No package installed."
msgstr ""

#: ../dnf/cli/commands/__init__.py:988 ../dnf/cli/commands/__init__.py:1109
#: ../dnf/cli/commands/__init__.py:1165
msgid "No package installed from the repository."
msgstr ""

#: ../dnf/cli/commands/__init__.py:1248
msgid "Error: Requires at least one package specification"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1271
msgid "Run commands on top of all packages in given repository"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1274
msgid "REPO"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1274
msgid "ARG"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1324
msgid "Error: Requires a repo ID and a valid sub-command"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1351
msgid "Display a helpful usage message"
msgstr "役立つ使い方のメッセージを表示する"

#: ../dnf/cli/commands/__init__.py:1352
msgid "COMMAND"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1384
#, python-format
msgid "No help available for %s"
msgstr "%s のヘルプは利用できません"

#: ../dnf/cli/commands/__init__.py:1389
msgid ""
"\n"
"\n"
"aliases: "
msgstr "\n\n別名: "

#: ../dnf/cli/commands/__init__.py:1391
msgid ""
"\n"
"\n"
"alias: "
msgstr "\n\n別名: "

#: ../dnf/cli/commands/__init__.py:1408
msgid "Display, or use, the transaction history"
msgstr "トランザクション履歴を表示、使用する"

#: ../dnf/cli/commands/__init__.py:1418
#, python-format
msgid ""
"Cannot undo transaction %s, doing so would result in an inconsistent package"
" database."
msgstr ""

#: ../dnf/cli/commands/__init__.py:1422
#, python-format
msgid ""
"Cannot rollback transaction %s, doing so would result in an inconsistent "
"package database."
msgstr ""

#: ../dnf/cli/commands/__init__.py:1504
msgid "Transactions:"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1505
msgid "Begin time  :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1506
msgid "End time    :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1507
msgid "Counts      :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1508
msgid "  NEVRAC :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1509
msgid "  NEVRA  :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1510
msgid "  NA     :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1511
msgid "  NEVR   :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1512
msgid "  rpm DB :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1513
msgid "  yum DB :"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1532
#, python-format
msgid "Unrecognized options \"%s\"!"
msgstr ""

#: ../dnf/cli/commands/__init__.py:1549
#, python-format
msgid "Invalid history sub-command, use: %s."
msgstr "使用した history のサブコマンドが正しくありません: %s"

#: ../dnf/cli/commands/__init__.py:1555
msgid "You don't have access to the history DB."
msgstr "履歴 DB にアクセスできません。"

#: ../dnf/base.py:111
#, python-format
msgid "%s, disabling."
msgstr ""

#: ../dnf/base.py:293
#, python-format
msgid "Warning: failed loading '%s', skipping."
msgstr ""

#: ../dnf/base.py:308
#, python-format
msgid "Repository %r: Error parsing config: %s"
msgstr "リポジトリー %r: 設定の解析中にエラー: %s"

#: ../dnf/base.py:314
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr "リポジトリー %r は構成中に名前がありませんので ID を使います"

#: ../dnf/base.py:364
#, python-format
msgid "Invalid tsflag in config file: %s"
msgstr "構成ファイルの tsflag が不正です: %s"

#: ../dnf/base.py:418
#, python-format
msgid "Failed to add groups file for repository: %s - %s"
msgstr "リポジトリーのグループファイルに追加できませんでした: %s - %s"

#: ../dnf/base.py:515
msgid "--> Finding unneeded leftover dependencies"
msgstr ""

#: ../dnf/base.py:571
msgid "Running transaction check"
msgstr ""

#: ../dnf/base.py:580
msgid "ERROR You need to upgrade rpm to handle:"
msgstr ""

#: ../dnf/base.py:582
msgid "ERROR with transaction check vs depsolve:"
msgstr ""

#: ../dnf/base.py:588
msgid "RPM needs to be upgraded"
msgstr ""

#: ../dnf/base.py:592
msgid "Transaction check succeeded."
msgstr ""

#: ../dnf/base.py:597
msgid "Running transaction test"
msgstr ""

#: ../dnf/base.py:609
msgid "Transaction check error:\n"
msgstr ""

#: ../dnf/base.py:616
msgid "Transaction test succeeded."
msgstr ""

#: ../dnf/base.py:633
msgid "Running transaction"
msgstr ""

#: ../dnf/base.py:708
msgid ""
"Warning: scriptlet or other non-fatal errors occurred during transaction."
msgstr "警告: スクリプト、もしくはその他で処理の間に致命的ではないエラーが発生しました。"

#: ../dnf/base.py:712
msgid "Transaction couldn't start (no root?)"
msgstr ""

#: ../dnf/base.py:713 ../dnf/base.py:724
msgid "Could not run transaction."
msgstr "トランザクションを実行できません。"

#: ../dnf/base.py:721
msgid "Transaction couldn't start:"
msgstr "トランザクションを解しできません:"

#: ../dnf/base.py:733
#, python-format
msgid "Failed to remove transaction file %s"
msgstr "トランザクションファイル %s の削除に失敗しました"

#: ../dnf/base.py:792
#, python-format
msgid "%s was supposed to be installed but is not!"
msgstr "%s のインストールを想定したがそうではなかった!"

#: ../dnf/base.py:852
#, python-format
msgid "%s was supposed to be removed but is not!"
msgstr "%s の削除を想定したがそうではなかった!"

#: ../dnf/base.py:956
#, python-format
msgid "Public key for %s is not installed"
msgstr "%s の公開鍵がインストールされていません"

#: ../dnf/base.py:960
#, python-format
msgid "Problem opening package %s"
msgstr "パッケージ %s を開いている最中に問題です"

#: ../dnf/base.py:968
#, python-format
msgid "Public key for %s is not trusted"
msgstr "%s の公開鍵が信頼されません"

#: ../dnf/base.py:972
#, python-format
msgid "Package %s is not signed"
msgstr "パッケージ %s は署名されていません"

#: ../dnf/base.py:1006
#, python-format
msgid "Cannot remove %s"
msgstr "%s を削除できません"

#: ../dnf/base.py:1010
#, python-format
msgid "%s removed"
msgstr "%s を削除しました"

#: ../dnf/base.py:1047
msgid "The enabled repos were expired"
msgstr ""

#: ../dnf/base.py:1076
#, python-format
msgid "Cannot remove %s file %s"
msgstr "%s のファイル %s を削除できませんでした"

#: ../dnf/base.py:1080
#, python-format
msgid "%s file %s removed"
msgstr "%s のファイル %s を削除しました"

#: ../dnf/base.py:1415
#, python-format
msgid "Adding package %s from group %s"
msgstr "パッケージ %s をグループ %s から追加しています"

#: ../dnf/base.py:1426
#, python-format
msgid "Warning: Group %s does not have any packages."
msgstr ""

#: ../dnf/base.py:1464
#, python-format
msgid "Package %s is already installed, skipping."
msgstr ""

#: ../dnf/base.py:1582
#, python-format
msgid "No package %s installed."
msgstr ""

#: ../dnf/base.py:1717
#, python-format
msgid "Failed to downgrade: %s"
msgstr "ダウングレードに失敗: %s"

#: ../dnf/base.py:1734
#, python-format
msgid "Failed to upgrade: %s"
msgstr ""

#: ../dnf/base.py:1858
#, python-format
msgid "Retrieving key from %s"
msgstr "%s から鍵を取得中です。"

#: ../dnf/base.py:1868
msgid "GPG key retrieval failed: "
msgstr "GPG 鍵の取得に失敗しました: "

#: ../dnf/base.py:1889
#, python-format
msgid "GPG key signature on key %s does not match CA Key for repo: %s"
msgstr "キー %s のGPGキーの署名は、次のレポジトリーのCA鍵と一致していません。: %s"

#: ../dnf/base.py:1892
msgid "GPG key signature verified against CA Key(s)"
msgstr ""

#: ../dnf/base.py:1900
#, python-format
msgid "Invalid GPG Key from %s: %s"
msgstr "%s からの GPG 鍵が正しくありません: %s"

#: ../dnf/base.py:1909
#, python-format
msgid "GPG key parsing failed: key does not have value %s"
msgstr "GPG 鍵の解析に失敗しました: 鍵は値 %s を持っていません"

#: ../dnf/base.py:1926
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" Package    : %s (%s)\n"
" From       : %s"
msgstr ""

#: ../dnf/base.py:1935
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" From       : %s"
msgstr ""

#: ../dnf/base.py:1963
#, python-format
msgid ""
"\n"
"\n"
"\n"
" Failing package is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr ""

#: ../dnf/base.py:1977
#, python-format
msgid "GPG key at %s (0x%s) is already installed"
msgstr "GPG 鍵 %s (0x%s) はすでにインストールしています"

#: ../dnf/base.py:2014
#, python-format
msgid "Key import failed (code %d)"
msgstr "鍵のインポートに失敗しました (コード: %d)"

#: ../dnf/base.py:2016 ../dnf/base.py:2117
msgid "Key imported successfully"
msgstr "鍵のインポートに成功しました"

#: ../dnf/base.py:2020
msgid "Didn't install any keys"
msgstr ""

#: ../dnf/base.py:2023
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct for this package.\n"
"Check that the correct key URLs are configured for this repository."
msgstr ""

#: ../dnf/base.py:2033
msgid "Import of key(s) didn't help, wrong key(s)?"
msgstr "鍵のインポートを助けられません。鍵が壊れていませんか?"

#: ../dnf/base.py:2055
msgid "No"
msgstr ""

#: ../dnf/base.py:2057
msgid "Yes"
msgstr ""

#: ../dnf/base.py:2058
#, python-format
msgid ""
"\n"
"\n"
"\n"
" CA Key: %s\n"
" Failing repo is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr ""

#: ../dnf/base.py:2071
#, python-format
msgid "GPG key at %s (0x%s) is already imported"
msgstr "GPG 鍵 %s (0x%s) はすでにインポートしています"

#: ../dnf/base.py:2115
#, python-format
msgid "Key %s import failed"
msgstr ""

#: ../dnf/base.py:2132
#, python-format
msgid "Didn't install any keys for repo %s"
msgstr ""

#: ../dnf/base.py:2137
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct.\n"
"Check that the correct key URLs are configured for this repository."
msgstr ""

#: ../dnf/conf.py:71
#, python-format
msgid "Could not set cachedir: %s"
msgstr "cackedir が設定できません: %s"

#: ../dnf/transaction.py:209
#, python-format
msgid "Will not install a source rpm package (%s)."
msgstr ""

#: ../dnf/plugin.py:121
#, python-format
msgid "Failed loading plugin: %s"
msgstr ""
